src/Pages/yongseok/Login/Login.js

```diff
@@ -1,13 +1,113 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import '../Common.scss';
+import './Login.scss';
 
 class Login extends Component {
-    render() {
-        return (
-            <div>
-                Hello
-            </div>
-        );
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+      emailValidateText: '',
+      passwordValidateText: '',
+      activate: false,
+      buttonActive: true,
+    };
+
+    this.emailRef = React.createRef();
+    this.passwordRef = React.createRef();
+  }
+
+  handleEmailValidate = () => {
+    const email = this.emailRef.current.value;
+    const regExp = /^[-A-Za-z0-9_]+[-A-Za-z0-9_.]*[@]{1}[-A-Za-z0-9_]+[-A-Za-z0-9_.]*[.]{1}[A-Za-z]{2,5}$/;
+    if (regExp.test(email)) {
+      this.setState({ email, emailValidateText: '' });
+      this.state.password &&
+        this.setState({ activate: true, buttonActive: false });
+    } else if (email.length === 0) {
```

Comment:
boolean값을 이용해 아래와 같이 작성해주실 수 있습니다 
```suggestion
    } else if (!email) {
```

Reply:
ㅜㅜ 제 코드 중에 === 0 이 너무 많았네요 ㅜㅜ 감사합니다 !! 수정완료 했습니다 !!